tests/ctst/features/iam-policies/AssumeRoleWithWebIdentity.feature

```diff
@@ -0,0 +1,144 @@
+Feature: Assume Role with Web Identity
+    In order to interact with restricted APIs
+    As an Artesca User
+    I want to use a web identity
+    And to subscribe to events I want to be notified on
+    And to receive notifications on buckets/objects activities I have subscribed to
+
+  @2.6.0
+  @PreMerge
+  @IAM-Policies-ARWWI
+  Scenario Outline: Assume Role with Web Identity
+    Given an existing bucket "<bucketName>" "<withVersioning>" versioning
+    And we "<haveObject>" an existing object "<objectName>" "<withVersionId>" version-Id "<objectVersionId>"
```

Comment:
As discussed in Xin's PR, I would suggest not specifying the name of entities we create, unless we really want to test them: the names **must** be unique for each test and managed by the code only. This will let us run CTST with multiple workers to speed up the testing time, if needed.
The only aspect we can alter is the entities existence (e.g., if the bucket is versioned or not)

```suggestion
    And an object that "<exists>" "<with>" version-Id"
```

Where `exists` can be "does not exist" or "exist" , and `with` can be "with" or "without", then the steps definition can be generic and re-usable easily, and we can also be consistent across scenarios

Reply:
if you really need to name the entities yourself, you can leverage the use of data tables. It was not done untill now so not tested as well, but this is supported by the framework that powers CTST: CucumberJS. A nice article to have examples : https://vinodmuralidharan.medium.com/cucumber-js-data-table-guide-a2447b74c86e